mcx/jax.py

```diff
@@ -109,7 +115,7 @@ def pbar_factory(func):
 
         def wrapper_progress_bar(carry, x):
             iter_num = x[0]
-            iter_num = _progress_bar((iter_num + 1, num_samples, print_rate), iter_num)
```

Comment:
I removed when using tqdm as progress bar as this sometimes caused the progress bar to stop at 90%. I'm not sure why though! Removing it seems to stop it from doing that